### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ktquez**
The package was a little out of date, with some changes and issues open.

The new version changes the vue-Announcer API a little to allow customization of politeness and cover other scenarios.

- [x] Update packages
- [x] Update rollup build files
- Custom politeness 
  - [x] At installation
  - [x] In meta params
  - [x] In set method
- [x] non-announce on a specific route using "skip"
- [x] Announce the correct landing page title (now supported on Vuepress)
- [x] Update typescript support to new API
- [x] Update cypress test
- [x] Update README
- [x] Documentation (Vuepress)
   - [x] Guide
   - [ ] Demos
   - [x] Browser testing [WIP]

Everyone's contribution will be perfect, it can be in the code, testing (e2e, unit) or even testing on different devices and software.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,17 +2,23 @@ module.exports = {
...
-    'standard',
-    'plugin:vue/essential'
+    'plugin:vue/recommended',
+    '@vue/standard',
+    'plugin:vue-a11y/base'
   ],
   plugins: [
-    "cypress"
+    'cypress',
+    'vue-a11y'
```
Comment (kddeisz) [inline .eslintrc.js:15]: Want to replace this with the new linter?

**Comment by ktquez** [comment]
Hello everybody, How are you?

Someone could help with the documentation, it could be creating demos, testing with assistive technology, or correcting English (not my native language).

The documentation of "next" (branch) is at this link https://announcer.vue-a11y.com/.

I would also like suggestions for API, if the new features are good or if there is something else that we could include to make the plugin even easier and more comprehensive.

Thank you all.

**Comment by kddeisz** [comment]
@ktquez it looks like the SSL certificate is invalid on that URL. Anything I can do to help get that fixed?

File: .eslintrc.js (hunk 1 above)
Comment (ktquez) [reply .eslintrc.js:15]: Yes, well-remembered!

**Comment by ktquez** [comment]
@kddeisz 
It's true, I forgot that custom domains in surge.sh free account don't force https, thanks for remembering.

I changed the link above, in the future we will migrate everything to subdomains on vue-a11y.com

File: .eslintrc.js (hunk 1 above)
Comment (ktquez) [reply .eslintrc.js:15]: Do you want to update?

**Pull request merged**